src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -2759,45 +2815,71 @@ subroutine atm_recover_large_step_variables_work(nCells, nEdges, nCellsSolve, nE
       integer :: i, iCell, iEdge, k, cell1, cell2
       real (kind=RKIND) :: invNs, rcv, p0, flux
 
+      MPAS_ACC_TIMER_START('atm_recover_large_step_variables [ACC_data_xfer]')
+      !$acc enter data copyin(rho_p_save,rho_pp,rho_base,rw_save,rw_p, &
+      !$acc                   rtheta_p_save,rtheta_pp,rtheta_base, &
+      !$acc                   ru_save,ru_p) &
+      !$acc            create(rho_zz,rho_p,rw,w,rtheta_p,theta_m, &
+      !$acc                   ru,u) &
+      !$acc            copyin(wwAvg,ruAvg)
```

Comment:
Out of curiosity, is there a reason for placing `wwAvg` and `ruAvg` in their own `copyin` clause rather than including them in the earlier `copyin` clause?

Reply:
Nothing I remember now. This is addressed in fixup commit [f631856](https://github.com/MPAS-Dev/MPAS-Model/pull/1220/commits/f631856105c78079ced1b347f95e5415cb270592)